### PR TITLE
doc(README): clearify how to continue after `sw360chores.pl -- up`

### PR DESCRIPTION
**Pull request by maxhbr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# SW360 chores
+
+This repository contains code which sets up a Docker based deployment and development infrastructure for [SW360](https://github.com/eclipse/sw360).
+Is simplifies and abstracts the configuration.
```
Comment (bs-jokri) [inline README.md]: It

File: README.md [hunk 2]
```diff
@@ -26,6 +32,12 @@ To build all images and start them simply use
 ```
 ./sw360chores.pl --build -- up
 ```
+To get a full configured [SW360](https://github.com/eclipse/sw360) running, you need to compile the wars and place them into `./_deploy`.
```
Comment (bs-jokri) [inline README.md]: fully

File: README.md [hunk 3]
```diff
@@ -26,6 +32,12 @@ To build all images and start them simply use
...
+This can be done from within the SW360 source folder with a single command via
```
Comment (bs-jokri) [inline README.md]: SW360 project root instead of source folder?

**Review by bs-jokri — CHANGES_REQUESTED** [review]
could fix then i can merge

**Comment by maxhbr** [comment]
@bs-jokri done!

**Pull request merged**